Lib/pydoc.py

```diff
@@ -2059,7 +2059,7 @@ def interact(self):
         while True:
             try:
                 request = self.getline('help> ')
-                if not request: break
+                if not request.strip(): continue
             except (KeyboardInterrupt, EOFError):
                 break
             request = request.strip()
```

Comment:
Since we're calling `.strip()` a few lines down anyway, I think we should just use that instead of calling it again here.

```suggestion
            except (KeyboardInterrupt, EOFError):
                break
            request = request.strip()
            if not request:
                continue
```